lib/task/import/csvCheckImportTask.class.php

```diff
@@ -189,6 +215,43 @@ public function execute($arguments = array(), $options = array())
       }
     }
 
+    // Check that parent IDs actually exist if legacy and parent IDs were found
+    if (count($legacyIds) && count($parentIds))
+    {
+      $missingParentIds = array();
+
+      foreach ($parentIds as $parentId => $occurrences)
+      {
+        // If a legacy ID doesn't exist for a parent ID, then add to missing count
+        if (empty($legacyIds[$parentId]))
+        {
+          $missingParentIds[$parentId] = $occurrences;
+        }
+      }
+
+      print "\n\nMissing parent IDs:\n";
+      print "-------------------\n\n";
+
+      print sprintf("Total unique parent IDs: %d\n\n", count($parentIds));
+
+      print sprintf("Number of unique missing parent IDs: %d\n", count($missingParentIds));
+
+      // Display details about each missing parent and total number of rows affected
+      if (count($missingParentIds))
+      {
+        $orphans = 0;
+
+        foreach ($missingParentIds as $parentId => $numberOfChildren)
+        {
+          print sprintf("* %d (%d children)\n", $parentId, $numberOfChildren);
```

Comment:
The output from this is:

```
* 10000 (1 children)
```

I think it would be helpful to indicate the first number is the parentId value, e.g.

```
* parentId: 10000 (1 children)
```